x-pack/test/security_solution_api_integration/test_suites/detections_response/telemetry/trial_license_complete_tier/usage_collector/detection_rules.ts

```diff
@@ -48,6 +51,31 @@ import { deleteAllExceptions } from '../../../../lists_and_exception_lists/utils
 
 import { FtrProviderContext } from '../../../../../ftr_provider_context';
 
+/**
+ * Helper to assert the expected values for elastic_total, elastic_customized_total, and elastic_noncustomized_total
+ */
+export function expectElasticTotals(
+  usage: RulesTypeUsage,
+  metric: keyof FeatureTypeUsage,
+  expected: { total: number; customized: number; noncustomized: number }
+) {
+  expect(usage.elastic_total[metric]).to.eql(expected.total);
+  expect(usage.elastic_customized_total[metric]).to.eql(expected.customized);
+  expect(usage.elastic_noncustomized_total[metric]).to.eql(expected.noncustomized);
+}
+
+/**
+ * Helper to assert that elastic_total = elastic_customized_total + elastic_noncustomized_total for all main metrics
+ */
+export function expectElasticSumBasicInvariant(usage: RulesTypeUsage) {
+  expect(usage.elastic_total.enabled).to.eql(
+    usage.elastic_customized_total.enabled + usage.elastic_noncustomized_total.enabled
+  );
+  expect(usage.elastic_total.disabled).to.eql(
+    usage.elastic_customized_total.disabled + usage.elastic_noncustomized_total.disabled
+  );
+}
+
```

Comment:
Shared functions shouldn't be part of a test file. We have `x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/telemetry` folder for shared functions.

Another concern is that function names don't fully reflect function bodies. It's possible to get rid of the functions by moving the tests relevant to the prebuilt rules to a separate file and using much more flexible Jest's expect for assertions.